src/components/registrationPage/registrationPage.js

```diff
@@ -63,6 +64,11 @@ const RegistrationPage = () => {
             </div>
             <div className="left-panel left-panel-reg">
                 <h1 className="logo">LOGO</h1>
+            <div className="img-wrap-right">
+                <div className="img-wraped-right">
+                    <img src={bgimg} alt="bg"/>
+                </div>
+            </div>
```

Comment:
Hawing img element wrapped in 2 divs is redundant. If we'll need to have them wrapped later we'll do it later
 ```suggestion
            <div className="img-wrap-right">
                    <img src={bgimg} alt="bg"/>
            </div>
```